CONTRIBUTING.md

```diff
@@ -20,22 +20,28 @@ If editing README.md, please conform to the [standard readme specification](http
 
 ### PR Process
 
-Active development of `go-fil-markets` occurs on the `development` branch. All PRs should be made to the `development` branch, which is the default branch on Github.
+Active development of `go-fil-markets` occurs on the `master` branch. All PRs should be made to the `master` branch, which is the default branch on Github.
 
-Before a PR can be merged to `development`, it must:
+Before a PR can be merged to `master`, it must:
 1. Pass continuous integration.
-1. Be rebased and up to date with the development branch
+1. Be rebased and up to date with the `master` branch
 1. Be approved by at least two maintainers
 
-When merging normal PRs to development, always use squash and merge to maintain a linear commit history.
+When merging normal PRs to `master`, always use squash and merge to maintain a linear commit history.
 
 ### Release Process
 
-The `master` branch is consider our production branch, and should always be up to date with the latest tagged release.
+When creating a new full release, branch off master with a branch named release/*version-number*, where *version-number* is the ultimate tag you intend to create.
```

Comment:
Could you note whether `version-number` should start with a `v`? I think go and/or github do special things in that case (but I'm not sure of details). In any case, whether or not to `v` is one of the many small things that there is value aligning on throughout the project.